hadoop-common-project/hadoop-common/src/main/resources/core-default.xml

```diff
@@ -4468,4 +4468,14 @@ The switch to turn S3A auditing on or off.
     <description>The AbstractFileSystem for Ozone FileSystem o3fs uri</description>
   </property>
 
+  <property>
+    <name>fs.$SCHEME.impl.disable.cache</name>
```

Comment:
I don't think we need this, $SCHEME won't resolve to anything & this config won't be used anywhere

Reply:
@ayushtkn Sir, thanks a lot for reviewing.  A colleague asked me about this config and he could not find any description at https://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-common/core-default.xml .  The reason of adding this config is providing a reference in website link for end-user.  Hope to receive your response again.

Reply:
I see, but I don't see any config in that doc which is a dummy config just for the sake of documentation, so maybe we can document it somewhere but not here

Reply:
Got it. Thanks Sir.

Reply:
How about make $SCHEME to hdfs ? Just like the `fs.har.impl.disable.cache` config in the docs.

![image](https://github.com/user-attachments/assets/93af7546-7f3f-4413-879b-0463161d1843)